src/gmt_api.c

```diff
@@ -507,12 +507,12 @@ GMT_LOCAL unsigned int api_gmtry (unsigned int geometry) {
 
 GMT_LOCAL inline struct GMTAPI_CTRL * api_get_api_ptr     (struct GMTAPI_CTRL *ptr)  {return (ptr);}
```

Comment:
Are you going to rename other functions e.g. `api_get_api_ptr` in future PRs?

Reply:
Yes, and there are a few that requires special checking such as the ones we build into gmt_core_module.c etc from the scripts.  This is just the bulk with more to come.